config/app_local.php.template

```diff
@@ -259,6 +259,8 @@ return [
             // Interlingua
             'ia' => ['Interlingua'], 'ina' => 'ia',
             // Italian
+            'is' => ['íslenska'], 'isl' => 'is',
+            // Icelandic
```

Comment:
The comments for Italian and Icelandic should be swapped.

In this line it should be `// Italian` and `// Icelandic` should be where `// Italian` is.